containers/server-systemd-services/uyuni-server.service

```diff
@@ -0,0 +1,37 @@
+[Unit]
+Description=Uyuni server image container service
+Wants=network.target
+After=network-online.target
+After=postgresql.service
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+EnvironmentFile=/etc/sysconfig/uyuni-server-systemd-services
+Restart=on-failure
+ExecStart=/usr/bin/podman run --tmpfs /run \
+    -p 443:443 -p 80:80 -p 4505:4505 -p 4506:4506 \
```

Comment:
you miss a few ports here. At least UDP 69 and TCP 25151, 5432 (report db) and possibly 1232 and 1233